README.md

```diff
@@ -118,6 +118,68 @@ export class AppComponent {
 }
 ```
 
+`SortBy` - sort an array by a specific property in individual elements
+```js
+import { arraySort } from 'helpful-decorators';
+
+class Test {
+  
+  @SortBy('name', {
+    isDescending: false,
+    type: 'string'
+  })
+  testingArrayAscending = [
+    {
+      name: 'b'
+    },
+    {
+      name: 'a'
+    },
+    {
+      name: 'c'
+    }
+  ];
+  /* Output:
```

Comment:
Can we make the readme cleaner, please? There is no need to show the output. And change everything to this style:
```ts
 @SortBy('name', {
    isDescending: false,
    type: 'string'
  })
  names = [{ name: 'a'}, { name: 'b' }]
```

Reply:
Hi @NetanelBasal ,

Thanks for the feedback, it makes sense to keep README simple and clean.

README is updated and it shows how to use it and the decorator is capable of sort **date**, **string** & **number**. 

Please feel free to leave more feedback.

Greetings from Yuchao :)